velox/substrait/SubstraitToVeloxPlan.cpp

```diff
@@ -1745,6 +1745,12 @@ void SubstraitVeloxPlanConverter::setFilterMap(
       }
       setColInfoMap<double>(functionName, colIdxVal, val, reverse, colInfoMap);
       break;
+    case TypeKind::BOOLEAN:
```

Comment:
Any chance to add some tests?

Reply:
Yes, actually we lack the tests for all pushed down filters converted from substrait. We plan to have a full tests for filters part in another PR. Thanks!